src/Windows/Avalonia.Win32/Win32Platform.cs

```diff
@@ -126,14 +125,7 @@ public IDisposable StartTimer(DispatcherPriority priority, TimeSpan interval, Ac
                 (uint)interval.TotalMilliseconds,
                 timerDelegate);
 
-            // Prevent timerDelegate being garbage collected.
-            _delegates.Add(timerDelegate);
```

Comment:
The `SafeHandle` is susceptible to GC as well. Is it enough that it's being held by the `DispatcherTimer`? Why was this delegate array needed in the first place?

Let's say I do `new DispatcherTimer()` and not hold onto its reference. I might expect it to continue ticking... This is how it would've behaved in WPF since timers were rooted by the Dispatcher, I think. `System.Timers.Timer` and `System.Threading.Timer` also behave this way.